skin/src/main/java/org/researchstack/skin/AppPrefs.java

```diff
@@ -24,11 +24,16 @@
         prefs = PreferenceManager.getDefaultSharedPreferences(context);
     }
 
-    public static synchronized AppPrefs getInstance(Context context)
+    public static void init(Context context) {
```

Comment:
getInstance() is synchronized, and it doesn't seem the instance variable is used in a thread safe manner in this class